test/system/030-run.bats

```diff
@@ -1006,6 +1006,20 @@ EOF
     CONTAINERS_CONF="$containersconf" run_podman 1 run --rm --read-only-tmpfs=false $IMAGE touch /tmp/testro
 }
 
+@test "podman run ulimit from containers.conf" {
+    containersconf=$PODMAN_TMPDIR/containers.conf
+    nofile=$(random_int 4)
+    cat >$containersconf <<EOF
+[containers]
+default_ulimits = [
+  "nofile=${nofile}:${nofile}",
+]
+EOF
+
+    CONTAINERS_CONF="$containersconf" run_podman run --rm $IMAGE grep "Max open files" /proc/self/limits
+    assert "$output" =~ "${nofile}.*${nofile}"
```

Comment:
Spaces, please. Just to make sure we don't falsely pass if `nofile=5` and the limits file shows `5500`. (Not like that's even possible, but paranoia is a virtue).
```bash
    assert "$output" =~ " ${nofile}  * ${nofile}  * files"
```
(Please copy-paste, because the multiple spaces are important but not obvious to the eye).